### DISCUSSION TITLE
can i access programms from another docker container?

### DISCUSSION THREAD

**Discussion by pcace** [caprover/caprover#1672] [General]
Hi there, 

is it possible, to access programms from one app to another? 
so if i install a docker container as an a app wich contains a programm i need in another app, is that possible (its not a webservice). 

cheers and thanks!

**Comment by githubsaturn** [comment]
Unless the service is accessible through a port, no, it's not possible. 

For example, the an app can access another container that is a database through a port. But if you are referring to running a command line program that is inside another container, no, it's not possible. Docker isolates the file system.

**Reply by pcace** [reply]
thanks for your quick reply!
